src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -102,24 +108,31 @@ public Lift() {
     }
 
     protected double returnPIDInput() {
-        return liftEncoder.getDistance();
+        return liftHeight();
     }
 
     protected void usePIDOutput(double output) {
         if (topLimitSwitch.get() == true && output > 0) {
-            setSetpoint(LiftHeight.kMaxHeight.get());
+            setSetpoint(DesiredLiftHeight.kMaxHeight.get());
             liftDrive.stopMotor();
             return;
         }
         if (bottomLimitSwitch.get() == true && output < 0) {
             liftEncoder.reset();
-            setSetpoint(LiftHeight.kFloor.get());
+            setSetpoint(DesiredLiftHeight.kFloor.get());
             liftDrive.stopMotor();
             return;
         }
         liftDrive.set(output);
     }
 
+    public double liftHeight() {
+        if (liftLidar.getDistance(true) > 72) {
```

Comment:
What is 72 supposed to be?

Reply:
I believe this is supposed to be the maximum distance the LiDAR can read? In which case we probably want this to be lower. We can also probably do some sort of correlation between the LiDAR and the encoder instead of using only the LiDAR.